screens/map/MapScreen.js

```diff
@@ -252,7 +252,7 @@ export default class MapScreen extends React.Component {
             enabledInnerScrolling={false}
             enabledBottomClamp
             overdragResistanceFactor={1}
-            enabledGestureInteraction
+            enabledContentTapInteraction={false}
```

Comment:
Honestly, I don't understand why this has to be false for the buttons to be clickable. It seems super counterintuitive but it finally worked.